src/script/chem/molfile/molfile.js

```diff
@@ -386,6 +387,53 @@ Molfile.prototype.writeBondBlock3000 = function (molecule) {
 	this.writeCR('M  V30 END BOND');
 };
 
+/**
+ * @param {Struct} molecule
+ *
+ * Example collection block:
+ * 	M  V30 BEGIN COLLECTION
+ * 	M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+ * 	M  V30 MDLV30/STEREL4 ATOMS=(1 2)
+ * 	M  V30 MDLV30/STERAC5 ATOMS=(2 1 6)
+ * 	M  V30 END COLLECTION
+ */
+Molfile.prototype.writeCollectionBlock3000 = function (molecule) {
+	const { abs, rac, rel } = molecule.enhancedStereo;
+	this.writeCR('M  V30 BEGIN COLLECTION');
+
+	// Absolute stereochemistry collection
+	// eg: M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+	if (abs.length) {
+		const atomIndices = abs.map((id) => id + 1).join(' ');
+		let absoluteDetails = this.getMol3000Prefix();
+		absoluteDetails += `MDLV30/STEABS ATOMS=(${abs.length} ${atomIndices})`;
+		const absoluteDetailsLines = absoluteDetails.match(/.{1,70}/g);
+		this.writeCR(absoluteDetailsLines.join(' -\nM  V30 '));
+	}
+
+	// Relative stereochemistry collection
+	// eg: M  V30 MDLV30/STEREL4 ATOMS=(1 2)
+	for (let [relLabel, relIndices] of rel.entries()) {
+		const atomIndices = relIndices.map((id) => id + 1).join(' ');
+		let absoluteDetails = this.getMol3000Prefix();
+		absoluteDetails += `MDLV30/STEREL${relLabel} ATOMS=(${relIndices.length} ${atomIndices})`;
+		const absoluteDetailsLines = absoluteDetails.match(/.{1,70}/g);
+		this.writeCR(absoluteDetailsLines.join(' -\nM  V30 '));
+	}
+
+	// "Racemic" stereochemistry collection
+	// eg: M  V30 MDLV30/STERAC5 ATOMS=(2 1 6)
+	for (let [racLabel, racIndices] of rac.entries()) {
+		const atomIndices = racIndices.map((id) => id + 1).join(' ');
+		let absoluteDetails = this.getMol3000Prefix();
+		absoluteDetails += `MDLV30/STERAC${racLabel} ATOMS=(${racIndices.length} ${atomIndices})`;
+		const absoluteDetailsLines = absoluteDetails.match(/.{1,70}/g);
+		this.writeCR(absoluteDetailsLines.join(' -\nM  V30 '));
+	}
+
```

Comment:
this block is almost a perfect dup of the above block. Do you expect them to diverge in future, or can something be abstracted?

Reply:
@mycrobe , I don't think they'll diverge in the future. 
I believe we should work on code deduplication for "**writers**" through a later ticket once we have investigated on [SS-29875](https://jira.schrodinger.com/browse/SS-29875), unless there's a quick refactor here.